discovery-service/src/main/java/org/zowe/apiml/discovery/ApimlInstanceRegistry.java

```diff
@@ -265,23 +259,33 @@ public void register(InstanceInfo info, final boolean isReplication) {
      * The instanceId must follow the format 'hostname:serviceId:port'.
      * The serviceId extracted from the instanceId must match both appName.
      * If any of these checks fail, the method will throw an exception and the registration is rejected.
+     * For backwards compatibility prints warnings only.
      * @param info the instance info
-     * @throws MetadataValidationException exception if any service identifier is invalid or inconsistent
      */
-    private void isServiceIdConformant(InstanceInfo info) {
+    private void validateInstanceInfo(InstanceInfo info) {
         String instanceId = info.getInstanceId();
         String appName = StringUtils.lowerCase(info.getAppName());
 
-        EurekaUtils.validateServiceId(appName);
-        String serviceId = EurekaUtils.getServiceIdFromInstanceId(instanceId);
-
-        EurekaUtils.validateServiceId(serviceId);
+        try {
+            EurekaUtils.validateServiceId(appName);
+        } catch (MetadataValidationException e) {
+            log.warn("Conformance criteria violation in appName: {}", e.getMessage());
+            log.debug("Invalid instance info provided: {}. Cause: {}", info, e.getMessage());
```

Comment:
why the instance info one is in debug? wouldn't make sense to have it on warn too?